lib/mayaUsd/ufe/UsdUndoMaterialCommands.h

```diff
@@ -93,6 +96,9 @@ class MAYAUSD_CORE_PUBLIC UsdUndoAssignNewMaterialCommand : public Ufe::InsertCh
     UsdUndoAssignNewMaterialCommand(
         const UsdSceneItem::Ptr& parentItem,
         const std::string&       sdrShaderIdentifier);
+    UsdUndoAssignNewMaterialCommand(
+        const std::vector<const UsdSceneItem::Ptr>& parentItems,
```

Comment:
There is already a predefined type `std::list<std::shared_ptr<SceneItem> > SceneItemList;` that the code can perhaps use.

Reply:
Good one, thank you. Much more readable.